consutil/main.py

```diff
@@ -39,6 +40,10 @@ def show():
             pid, date = busyDevices[lineNum]
             busy = "*"
         baud = getBaud(lineNum)
+        if baud == "":
+            baud = "9600/-"
```

Comment:
I have a small concern here. We are hard coding 9600 as default baud rate. But the actual default value is either decided by piccocom or some other code in console util.

This creates a possibility for inconsistent default value in the future. Is it possible to unify the code where default value was chosen?

Reply:
Yeah, right now connect() also sets the default baud to 9600, so that default value is hardcoded in two different places. I can change getConnectionInfo() to return (actual_baud, conf_baud, flow_control) instead of just (baud_rate, flow_control)